trunk/source/Library-Automata/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,8 @@ Require-Bundle: de.uni_freiburg.informatik.ultimate.lib.core,
  de.uni_freiburg.informatik.ultimate.smtinterpol,
  de.uni_freiburg.informatik.ultimate.lib.smtlib,
  de.uni_freiburg.informatik.ultimate.lib.smtlibutils,
- de.uni_freiburg.informatik.ultimate.lib.util
+ de.uni_freiburg.informatik.ultimate.lib.util,
+ wrapped.net.sf.trove4j.trove4j
```

Comment:
Is trove actually used?

Reply:
Yes, see [here](https://github.com/ultimate-pa/ultimate/blob/0a57b278650c6e4f7c1dc59465c12955e1a35a6e/trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/nestedword/operations/optncsb/automata/AccGenBuchi.java#L33). I am however not sure, if this automaton operation is _actually_ used 😉